### ISSUE TITLE
Decomposition - cudnn_batch_norm vs native_batch_norm

### ISSUE THREAD

**Issue by anijain2305** [pytorch/pytorch#93787]
For few of the TIMM models, there are accuracy differences when I use AOT Autograd. On further debugging, I found that removing `cudnn_batch_norm` and `cudnn_batch_norm_backward` from the decomposition list resolves the issue. 

I am able to repro it, but in somewhat unusual way. I have incpetion model. In the first version, all the batch norm ops are `cudnn_batch_norm` ops. In the second version, I have replaced the very first `cudnn_batch_norm` with `native_batch_norm`. With this, I am able to repro the accuracy difference.

The zip file is here

[inception.tar.gz](https://github.com/pytorch/torchdynamo/files/9145065/inception.tar.gz)

The file - `inception_fail.py` compares the above two versions of the model and shows that the output tensors deviate a lot.

cc @ezyang @soumith @msaroufim @wconstab @ngimel @bdhirsh
